task02/task02/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите, пожалуйста, трёхзначное число:");
+            int numberABC = int.Parse(Console.ReadLine());
+            Console.WriteLine("Исходное число: " + numberABC);
+            int c = numberABC % 10;
+            int ab = numberABC / 10;
+            int b = ab % 10;
+            int a = ab / 10;
+            Console.WriteLine("Полученное число: " + c + b + a);
```

Comment:
В задании требовалось получить(т. е. **вычислить**) число, а не напечатать число. Вы выводите на консоль строку, в которой записано искомое число, но самого числа вы не вычисляете.
```suggestion
            var number = c * 100 + b * 10 + a;
            Console.WriteLine("Полученное число: " + number);
```

Reply:
Хорошо, понимаю, что надо было сделать. Спасибо, что поправили!